src/emulator/functionsEmulator.ts

```diff
@@ -672,6 +672,18 @@ export class FunctionsEmulator implements EmulatorInstance {
       disabled_features: this.args.disabledRuntimeFeatures,
     };
   }
+  /**
+   * Returns a node major version ("10", "8") or null
+   * @param frb the current Functions Runtime Bundle
+   */
+  getRequestedNodeRuntimeVersion(frb: FunctionsRuntimeBundle): string | null {
```

Comment:
Is there a reason this has to be `string | null` not `string | undefined`?  If you return `undefined` in the case of nothing then it's just:
```js
return frb.nodeMajorVersion || (pkg.engines && pkg.engines.node)
```

If you do need `null` then:
```js
const pkg = require(path.join(frb.cwd, "package.json"));
return frb.nodeMajorVersion || (pkg.engines && pkg.engines.node) || null
```

But this is all optional, method is clearly correct as-is just maybe overly complex?